src/library/data-functions/data-functions-geotools/src/main/java/com/camptocamp/opendata/producer/geotools/UnicodeSimpleFeaturePropertyAccessorFactory.java

```diff
@@ -0,0 +1,74 @@
+package com.camptocamp.opendata.producer.geotools;
+
+import java.util.regex.Pattern;
+
+import org.geotools.api.feature.IllegalAttributeException;
+import org.geotools.api.feature.simple.SimpleFeature;
+import org.geotools.api.feature.simple.SimpleFeatureType;
+import org.geotools.filter.expression.PropertyAccessor;
+import org.geotools.filter.expression.PropertyAccessorFactory;
+import org.geotools.util.factory.Hints;
+
+/**
+ * GeoTools {@link PropertyAccessorFactory} that allows to evaluate
+ * {@link SimpleFeatureType} and {@link SimpleFeature} attributes containing any
+ * unicode character, spaces, and punctuation signs in their names, as long as
+ * they start with a (unicode) letter.
+ */
+public class UnicodeSimpleFeaturePropertyAccessorFactory implements PropertyAccessorFactory {
+
+    public static final PropertyAccessor UNRESTRICTED_ATTRIBUTE_NAME_ACCESS = new UnrestrictedUnicodePropertyAccessor();
+
+    /**
+     * https://regex101.com/r/XCpNjr/1
```

Comment:
not sure this will be stable :) Maybe a more meaningful comment on what the following regex is about ?

Reply:
I think the explanation in the class javadoc is clear enough?
```
/**
 * GeoTools {@link PropertyAccessorFactory} that allows to evaluate
 * {@link SimpleFeatureType} and {@link SimpleFeature} attributes containing any
 * unicode character, spaces, and punctuation signs in their names, as long as
 * they start with a (unicode) letter.
 */
```